service/src/main/kotlin/fi/espoo/evaka/placement/PlacementQueries.kt

```diff
@@ -842,3 +842,39 @@ WHERE pl.child_id = ${bind(childId)} AND daterange(pl.start_date, pl.end_date, '
             )
         }
         .exactlyOneOrNull<Language>()
+
+fun Database.Transaction.cleanupFutureReservationsAndAbsencesOutsideValidPlacements(
```

Comment:
Perhaps the term should be `delete` instead of `cleanup` for better discoverability? We currently use `delete` and `clear` in names of functions that delete something from the database, `cleanup` would be a third term.

Reply:
good point, done